### PR TITLE
[RA1-Ch8] Multi-cloud gap

### PR DESCRIPTION
**Pull request by sukhdevkapur**
The addresses Issue #1836 
A suggestion to address the gap is documented for Baraque release

### CODE REVIEW

File: doc/ref_arch/openstack/chapters/chapter08.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ The most obvious way to address the gaps will be to propose a set of APIs in the
 
 * Public and Multi-Cloud  - The framework offered by OpenStack does not support multi-cloud deployments, along with the core deployments
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter08.md:20]: @sukhdevkapur and @walterkozlowski 

Do we first need to define the problem more clearly and as to what is meant by and needed for multi-cloud deployments? The needs maybe in terms of use cases or feature functionality.

Then, do we need to define a set of requirements in RM or in RA's for multi-cloud deployments?

Maybe then, in the RAs, we can specify how are we going to satisfy these requirements.

Critique?

**Comment by walterkozlowski** [comment]
@pgoyal01, I have already asked @sukhdevkapur to raise an Issue for RM about this gap, and we will need to address it as other gaps in RM Chapter 10.  I leave it to @pgoyal01 to decide whether he wants to leave the proposed comment by @sukhdevkapur  in RA-1.  It looked reasonable to me.

File: doc/ref_arch/openstack/chapters/chapter08.md [hunk 2]
```diff
@@ -19,6 +19,8 @@ The most obvious way to address the gaps will be to propose a set of APIs in the
...
 
+From the short term point of view, we could use ONAP to deal with this gap. However, for a long term point of view, reliance on ONAP should be avoided and the OpenStack APIs should be implemented/enhanced to address this gap.
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter08.md:22]: In the short-term, how does ONAP support multi-cloud deployments? Is it a central scheduler that decides which cloud the deployment should be made on? Is it capable of creating a multi-cloud overlay, if that is one of the needs of a multi-cloud deployment? Etc. etc.

In the long-term, what is the nature of the APIs that we are requesting for which OSTK services beyond Identity (Keystone) and Dashboard (Horizon).

Comment (karinesevilla) [reply doc/ref_arch/openstack/chapters/chapter08.md:22]: @sukhdevkapur Can ONAP be replaced by "an orchestrator such as ONAP" to make it more open?

**Comment by pgoyal01** [comment]
@walterkozlowski @sukhdevkapur @karinesevilla  Can we close this pull request as PR #1857 and PR #1858 supersede this?

**Comment by pgoyal01** [comment]
@walterkozlowski @sukhdevkapur 
Discussed in RA1 meeting of August 3, 2020:
PR #1857/1858: The content of RAI Ch 8.2 is an OpenStack multi-cloud support gap and doesn't belong in RM.  Ask that the PRs 1857 and 1858 be closed without any action. If RM needs to add multicloud support as a gap it should be a new issue.

**Comment by walterkozlowski** [comment]
Closed, RM will create its own PR if needed

**Pull request closed without merging**